tests/files/api_fixtures/gcp_set_subscription_admin_verified.yaml

```
@@ -0,0 +1,237 @@
+        "force_encryption": false,
+        "force_verify": false,
+        "french_english_bilingual": false,
+        "gcp_connected": true,
+        "gcp_paused": false,
+        "gcs_manager_url": null,
+        "gcs_version": null,
+        "globus_connect_setup_key": null,
+        "guest_collection_activity_notification_policy": null,
+        "high_assurance": false,
+        "host_endpoint": null,
+        "host_endpoint_display_name": null,
+        "host_endpoint_id": null,
+        "host_path": null,
+        "https_server": null,
+        "id": "11111111-1111-1111-1111-111111111111",
+        "in_use": false,
+        "info_link": null,
+        "is_globus_connect": true,
+        "keywords": null,
+        "last_accessed_time": null,
+        "local_user_info_available": null,
+        "location": "Automatic",
+        "mapped_collection_display_name": null,
+        "mapped_collection_id": null,
+        "max_concurrency": 4,
+        "max_parallelism": 4,
+        "mfa_required": false,
+        "my_effective_roles": [
+          "administrator",
+          "activity_manager",
+          "activity_monitor"
+        ],
+        "myproxy_dn": null,
+        "myproxy_server": "myproxy.globusonline.org",
+        "name": "11111111-1111-1111-1111-111111111111",
+        "network_use": "normal",
+        "non_functional": false,
+        "non_functional_endpoint_display_name": null,
+        "non_functional_endpoint_id": null,
+        "oauth_server": null,
+        "organization": null,
+        "owner_id": "75d82364-b765-4806-96dd-1bf335ae9b7a",
+        "owner_string": "nobody@globus.org",
+        "preferred_concurrency": 2,
+        "preferred_parallelism": 2,
+        "preserve_modtime": null,
+        "public": true,
+        "requester_pays": false,
+        "restrict_transfers_to_high_assurance": null,
+        "s3_owner_activated": false,
+        "s3_url": null,
+        "shareable": true,
+        "sharing_target_endpoint": null,
+        "sharing_target_root_path": null,
+        "skip_auto_delete": null,
+        "subscription_admin_verified": true,
+        "subscription_id": "aa4d684f-29d0-4b07-8a03-4f8cd570f982",
+        "tlsftp_server": null,
+        "user_message": null,
+        "user_message_link": null,
+        "username": "u_xhavtnb4b5g3jkh632ewsb7eeu"
+      }
+  - path: /endpoint/ffffffff-ffff-ffff-ffff-ffffffffffff
+    method: get
+    json:
+      {
+        "DATA": [
+          {
+            "DATA_TYPE": "server",
+            "hostname": null,
+            "id": 575,
+            "incoming_data_port_end": null,
+            "incoming_data_port_start": null,
+            "is_connected": false,
+            "is_paused": false,
+            "outgoing_data_port_end": null,
+            "outgoing_data_port_start": null,
+            "port": null,
+            "scheme": null,
+            "subject": "/C=US/O=Globus Consortium/OU=Globus Connect Service/CN=0845ac3e-4ba1-11f0-9567-0affd731719d",
```

Don't know if this trailing UUID is a Globus CNAME (which seems okay to remain) or if this CNAMEs are generated (which seems more likely to need to be obviously bogus). Perhaps:

```suggestion
            "subject": "/C=US/O=Globus Consortium/OU=Globus Connect Service/CN=00000000-4ba1-11f0-9567-0affd731719d",
```